lang/test/org/partiql/lang/TestBase.kt

```diff
@@ -36,7 +34,7 @@ abstract class TestBase : Assert() {
     val ion: IonSystem = IonSystemBuilder.standard().build()
     val valueFactory = ExprValueFactory.standard(ion)
 
-    val defaultRewriter = AstRewriterBase()
+    val defaultTransform = PartiqlAst.VisitorTransform()
```

Comment:
Probably aught to undo this particular change as this is one of the main ways that the `AstRewriterBase` is itself tested.  See other comment on `assertBaseTransform`.